ontologies/vocabs.ttl

```diff
@@ -434,8 +424,10 @@ Adversaries motivated by notoriety are often seeking either personal validation
 
 gist:_AttackMotivation_organizational-gain
 	a gist:AttackMotivation ;
-	rdfs:label "Oranizational Gain"^^xsd:string ;
-	skos:altLabel "Oranizational-Gain"^^xsd:string ;
+	skos:altLabel
+		"Oranizational Gain"^^xsd:string ,
+		"Oranizational-Gain"^^xsd:string
```

Comment:
Is this helpful?
Also, should there not be a stixTerm? I guess that is a different git issue?